File: src/androidTest/java/pt/joaomneto/titancompanion/TestMOM.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;

import static pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.MASKS_OF_MAYHEM;

@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestMOM extends TestTFOD {

    @Override
    protected FightingFantasyGamebook getGamebook() {
        return MASKS_OF_MAYHEM;
    }

}

File: src/androidTest/java/pt/joaomneto/titancompanion/TestMOM.kt
package pt.joaomneto.titancompanion

import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.filters.LargeTest
import org.junit.runner.RunWith
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.MASKS_OF_MAYHEM

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestMOM : TestTFOD() {

    override val gamebook = MASKS_OF_MAYHEM
}
